packages/vx-geo/src/projections/Projection.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { Group } from '@vx/group';
+import additionalProps from '../util/additionalProps';
+import { geoOrthographic, geoAlbers, geoMercator, geoPath } from 'd3-geo';
+
+// TODO: Implement all projections of d3-geo
+const projectionMapping = {
+  orthographic: geoOrthographic(),
+  albers: geoAlbers(),
+  mercator: geoMercator()
+};
+
+/**
+ * Component for all projections.
+ */
+export default function Projection({
+  data,
+  projection = 'mercator',
+  projectionFunc,
+  clipAngle,
+  clipExtent,
+  scale,
+  translate,
+  center,
+  rotate,
+  precision,
+  fitExtent,
+  fitSize,
+  centroid,
+  className,
+  ...restProps
+}) {
+  const currProjection = projectionMapping[projection];
+
+  if (clipAngle) currProjection.clipAngle(clipAngle);
+  if (clipExtent) currProjection.clipExtent(clipExtent);
+  if (scale) currProjection.scale(scale);
+  if (translate) currProjection.translate(translate);
+  if (center) currProjection.translate(center);
+  if (rotate) currProjection.rotate(rotate);
+  if (precision) currProjection.rotate(precision);
+  if (fitExtent) currProjection.fitExtent(...fitExtent);
+  if (fitSize) currProjection.fitSize(...fitSize);
+
+  const path = geoPath().projection(currProjection);
+
+  return (
+    <Group className={`vx-${projection}-group`}>
+      {data.map((feature, i) => {
+        let c;
+        if (centroid) c = path.centroid(feature);
+        return (
+          <g key={`${projection}-${i}`}>
+            <path
+              className={classnames(`vx-${projection}`, className)}
+              d={path(feature)}
+              {...additionalProps(restProps, {
+                ...feature,
+                index: i,
+                centroid: c
+              })}
+            />
+            {centroid && centroid(c, feature)}
+          </g>
+        );
+      })}
+      {/* TODO: Maybe find a different way to pass projection function to use for example invert */}
```

Comment:
yeah it's a bummer there is no `projection.copy()` utility analogous to `scale`s 🤔